Makefile

```diff
@@ -319,132 +269,216 @@ jumpstart-opensuse-leap:
 	libXext6 libX11-6 libX11-devel libXt-devel openmotif-devel \
 	xterm espeak csh
 
+################################################################################
+# Download targets
+################################################################################
 .PHONY: download
-download: _download/Base.Downloaded.proxy _download/Corepops.Downloaded.proxy _download/Packages.Downloaded.proxy
-
-# Instructs the build process to assume that the sister github repos have been
-# cloned and/or downloaded and reside in ../Base etc. This does not
-# include Aaron Sloman's packages at this time. 
-.PHONY: use-repos
-use-repos: _download/Packages.Downloaded.proxy
-	mkdir -p _download/Corepops
-	( cd ../Corepops; tar cf - . ) | ( cd _download/Corepops; tar xf - )
-	mkdir -p _download/Base
-	( cd ../Base; tar cf - . ) | ( cd _download/Base; tar xf - )
-	touch _download/Base.Downloaded.proxy _download/Corepops.Downloaded.proxy
+download: _download/Corepops _download/Base _download/packages-V$(MAJOR_VERSION).tar.bz2 _download/poplogout.sh _download/poplogout.csh
+
+_download/Corepops:
+	mkdir -p "$@"
+	curl -LsS "$(COREPOPS_TARBALL_URL)" | ( cd _download/Corepops; tar zxf - --strip-components=1 )
+
+_download/Base:
+	mkdir -p "$@"
+	curl -LsS "$(BASE_TARBALL_URL)" | ( cd _download/Base; tar zxf - --strip-components=1)
+
+_download/packages-V$(MAJOR_VERSION).tar.bz2:
+	mkdir -p "$(@D)"
+	curl -LsS "http://www.cs.bham.ac.uk/research/projects/poplog/V$(MAJOR_VERSION)/DL/packages-V$(MAJOR_VERSION).tar.bz2" > "$@"
+
+_download/poplogout.%:
+	mkdir -p "$(@D)"
+	curl -LsS "https://www.cs.bham.ac.uk/research/projects/poplog/V$(MAJOR_VERSION)/DL/$(notdir $@)" > "$@"
+
+_download/appimagetool:
+	mkdir -p "$(@D)"
+	curl -LSs "https://github.com/AppImage/AppImageKit/releases/download/13/appimagetool-x86_64.AppImage" > "$@"
+	chmod a+x "$@"
+	[ -x $@ ] # Sanity check
+
+################################################################################
+# Source tarball targets
+################################################################################
+.PHONY: srctarball
+srctarball: $(SRC_TARBALL)
+
+$(SRC_TARBALL): _download/Corepops _download/Base _download/packages-V$(MAJOR_VERSION).tar.bz2 _download/poplogout.sh _download/poplogout.csh
+	mkdir -p "$(@D)"
+	rm -f "$@"; \
+	ASSEMBLY_DIR="$$(mktemp --directory --tmpdir="$(TMP_DIR)")"; \
```

Comment:
We need to to have control over the permissions that `mktemp` creates the directory with. This might be done with `umask`. There may be an option on `mktemp`.